### PR TITLE
Fix: Fixed an issue where the focus was lost when quickly switching tabs

### LINKED ISSUE #15288: Bug: Focus is lost when changing tabs
### Description

I use the keyboard all the time. I like to use the down arrow to, well, move down a file/folder list. I also use the ctrl-tab/shift-ctrl-tab to move between opened tabs. When doing so and when getting back to a tab where I was down a list, the "cursor" position is not maintained; further more, I cannot use the down arrow to move down unless I use the mouse to click where I want to be.

### Steps To Reproduce

1. Open two tabs
2. Move cursor to the center of the app window
3. Used `ctrl` + `tab` to switch to the second tab & `ctrl` + `shift` + `tab` to move back to the first tab
4. The cursor position moved

### Requirements

Keep cursor in the same position

### Files Version

3.1

### Windows Version

W11 23H2

### Log File

[debug.log](https://github.com/files-community/Files/files/15189392/debug.log)

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**

- Closes #15288

**Steps used to test these changes**

1. Changed tabs using Ctrl + Shift + Tab
2. Changed tabs using Ctrl + Tab
3. Changed tabs using Ctrl + 1..9

### CODE REVIEW

File: src/Files.App/Actions/Navigation/PreviousTabAction.cs [hunk 1]
```diff
@@ -26,14 +29,15 @@ public PreviousTabAction()
...
-		public Task ExecuteAsync(object? parameter = null)
+		public async Task ExecuteAsync(object? parameter = null)
 		{
 			if (App.AppModel.TabStripSelectedIndex is 0)
 				App.AppModel.TabStripSelectedIndex = multitaskingContext.TabCount - 1;
 			else
 				App.AppModel.TabStripSelectedIndex--;
 
-			return Task.CompletedTask;
+			await Task.Delay(500);
```
Comment (yaira2) [inline src/Files.App/Actions/Navigation/PreviousTabAction.cs:39]: Can you add a comment for future reference?

**Comment by yaira2** [comment]
@Josh65-2201 can you confirm if this fixes the issue for you?

**Comment by Josh65-2201** [comment]
Works for ctrl + shift + tab but doesn't for Ctrl + tab

**Comment by yaira2** [comment]
@Josh65-2201 should we move forward with this as it's a partial fix?

**Comment by Josh65-2201** [comment]
Yes

File: src/Files.App/Actions/Navigation/PreviousTabAction.cs (hunk 1 above)
Comment (d2dyno1) [reply src/Files.App/Actions/Navigation/PreviousTabAction.cs:40]: Done

**Comment by yaira2** [comment]
@Josh65-2201 can you check with the latest commit?

**Review by Josh65-2201 — APPROVED** [review]
Works for all three

**Comment by yaira2** [comment]
@d2dyno1 thank you!

**Pull request merged**